GoogleSignIn/Sources/Public/GoogleSignIn/GIDGoogleUser.h

```diff
@@ -53,6 +61,12 @@ NS_ASSUME_NONNULL_BEGIN
 /// see https://developers.google.com/identity/sign-in/ios/backend-auth.
 @property(nonatomic, readonly, nullable) GIDToken *idToken;
 
+/// A new authorizer for `GTLService`, `GTMSessionFetcher`, or `GTMHTTPFetcher`.
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+@property(nonatomic, readonly) id<GTMFetcherAuthorizationProtocol> fetcherAuthorizer;
```

Comment:
This could also be a small follow up project...if that's the right direction, let's add (and link together) issues here and on [GTMAppAuth](https://github.com/google/GTMAppAuth/issues).

Reply:
Thanks!